data/tlg5034/tlg001a/tlg5034.tlg001a.perseus-grc1.xml

```diff
@@ -48,20 +48,23 @@
                         <monogr>
                             <author>Scholia Pindar</author>
                             <title>Scholia in Pindarum Olympian Odes</title>
+                            <editor>A. B. Drachmann</editor>
                             <imprint>
                                 <pubPlace>Leipzig</pubPlace>
-                                <publisher>B.G. Teubneri</publisher>
+                                <publisher xml:lang="lat">B.G. Teubneri</publisher>
                                 <date>1903</date>
                             </imprint>
-                            <biblScope unit="volume">1</biblScope>
+                            
                         </monogr>
+                        <series><title level="s" xml:lang="lat">Scholia in Olympionicas</title><biblScope n="volume">Vol. 1</biblScope></series>
+                        
                         <ref type="https://archive.org/details/scholiaveterain00dracgoog">Internet Archive</ref>
                     </biblStruct>
                 </listBibl>
             </sourceDesc>
 
         </fileDesc>
-        <encodingDesc>
+        <encodingDesc> <p>text encoded in accordance with the latest EpiDoc standards</p>
               <refsDecl n="CTS">
                 <cRefPattern matchPattern="(.+).(.+)" n="line" replacementPattern="#xpath(/tei:TEI/tei:text/tei:body/tei:div/tei:div[@n='$1']/tei:div[@n='$2'])"/>
                 <cRefPattern matchPattern="(.+)" n="poem" replacementPattern="#xpath(/tei:TEI/tei:text/tei:body/tei:div/tei:div[@n='$1'])"/>
```

Comment:
Below in the `<langUsage>` element, add `<language ident="lat">Latin</language>`. We need this because we use the `xml:lang="lat"` attribute in different places in the header.